server/models/k8s_context.go

```diff
@@ -291,6 +306,16 @@ func (kc K8sContext) GenerateKubeHandler() (*kubernetes.Client, error) {
 	return kubernetes.New(cfg)
 }
 
+func (kc *K8sContext) AssignVersion(handler *kubernetes.Client) error {
```

Comment:
Is this function being used outside of package, if not, we can change it to be available on current scope only.
```suggestion
func (kc *K8sContext) assignVersion(handler *kubernetes.Client) error {
```